airbase/pom.xml

```diff
@@ -202,7 +202,7 @@
         <dep.assertj-core.version>3.26.3</dep.assertj-core.version>
         <dep.byte-buddy.version>1.15.1</dep.byte-buddy.version>
         <dep.checkstyle.version>10.18.1</dep.checkstyle.version>
-        <dep.errorprone.version>2.31.0</dep.errorprone.version>
+        <dep.errorprone.version>2.32.0</dep.errorprone.version>
```

Comment:
This version requires JDK 17 and airbase defaults to JDK 8 (`<project.build.targetJdk>8</project.build.targetJdk>`)
https://github.com/google/error-prone/issues/3803#issuecomment-2344601626

Reply:
Target is --release